code/modules/shuttle/mini_dropship.dm

```diff
@@ -173,6 +173,11 @@
 	open_prompt = FALSE
 	clean_ui_user()
 
+/obj/machinery/computer/camera_advanced/shuttle_docker/minidropship/attack_hand(mob/user)
```

Comment:
i haven't noticed before, but you are declaring a new proc inside another one? what is this?

Reply:
it's in the middle of another proc...